source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -237,6 +237,22 @@
             return valuesByKey;
         });
 
+        var commentsByKeyCollection = _.map(data.Califications, function (calification) {
+            var commentsByKey = {
+                commentRow: {
+                    calificationId: calification.Id,
+                    evaluatorEmployee: calification.EvaluatorEmployee,
+                    owner: calification.Owner
+                }
+            };
+            if (calification.Califications)
```

Comment:
could you add curly braces for this `if` to avoid possible issues in the future?